yarn-project/circuit-types/src/logs/tx_l2_logs.ts

```diff
@@ -99,8 +99,8 @@ export abstract class TxL2Logs<TLog extends UnencryptedL2Log | EncryptedL2Log> {
 }
 
 export class UnencryptedTxL2Logs extends TxL2Logs<UnencryptedL2Log> {
-  /** Creates an empty instance. */
-  public static empty() {
+  /** Creates an default instance. */
```

Comment:
Again, leaving further context in the comments could be beneficial. In this case, the default implementation is an empty one.